bin/package

```diff
@@ -9,7 +9,7 @@ mkdir -p dist
 
 pathspec="README.txt upload"
 version="$(git describe --tags --match 'v*')"
-out="$PWD/dist/opencart3-webwinkelkeur-$version.zip"
+out="$PWD/dist/opencart3-webwinkelkeur-$version.ocmod.zip"
```

Comment:
Extension Installer recognizes such zips and uploads their content directly to the FTP root of your store. That said, it will produce an error if you try to upload zips with a name that is not in the "***.ocmod.zip**" format.